include/lbann/layers/regularizers/dropout.hpp

```diff
@@ -65,15 +62,13 @@ class dropout : public regularizer_layer {
   dropout& operator=(const dropout& other) {
     regularizer_layer::operator=(other);
     m_keep_prob = other.m_keep_prob;
-    if(m_mask) {
-      delete m_mask;
-    }
+    if(m_mask) delete m_mask;
```

Comment:
Space, to be consistent.

Reply:
I believe that in the code style section we were avoiding if statements with no braces.